integrations/integrations.php

```diff
@@ -66,14 +68,12 @@ public function activate_platform_integrations() {
 			$vip_config = $this->get_integration_vip_config( $slug );
 
 			if ( $vip_config->is_active_via_vip() ) {
-				$this->activate( $slug, [
-					'config' => $vip_config->get_site_config(),
-				] );
+				$this->activate( $slug );
```

Comment:
No longer need to be passing in a "cached" version of `config` during platform activation, instead we'll just be pulling directly from the vip_config for these.